telegram/ext/filters.py

```diff
@@ -2545,3 +2546,60 @@ def filter(self, message: Message) -> bool:
 
 VOICE = _Voice("filters.VOICE")
 """Messages that contain :attr:`telegram.Message.voice`."""
+
+
+class Mention(MessageFilter):
+    """Mention messages. If list of username or chat_id or :attr:`telegram.User` objects passed,
+    it filters messages to only allow those whose contains one of username
+    or chat_id or :attr:`telegram.User`.
+    Note:
+        Usernames only allowed with @ prefix. Example: "@username"
```

Comment:
I just see that we discard leading '@' in all other filters that accept usernames, i.e. it doesn't matter if you pass the username with or without '@'. I would like to do the same here.

Reply:
I have fixed. I check all argument str, add '@'  if not added.